src/playground/src/ui/screen/popover.component.tsx

```diff
@@ -73,7 +73,8 @@ class Popover extends React.Component<Props, State> {
             style={component}
             placement={`${PLACEMENT} start`}
             visible={this.state.startVisible}
-            content={this.createPopoverContentElement('❤️')}>
+            content={this.createPopoverContentElement('❤️')}
```

Comment:
You have lost the heart :(

Reply:
It was just an extra heart. Why do you need one more than mine?